gobblin-hive-registration/src/main/java/gobblin/hive/publisher/BaseDataPublisherWithHiveRegistration.java

```diff
@@ -10,13 +10,14 @@
  * CONDITIONS OF ANY KIND, either express or implied.
  */
 
-package gobblin.publisher;
+package gobblin.hive.publisher;
```

Comment:
What's the purpose of the package change? Are we sure this won't break any external dependents?